Makefile

```diff
@@ -312,11 +326,11 @@ verify-manifests: manifests $(YQ) ## Verify manifests update.
 	yq e -e '.metadata.annotations.containerImage == "$(OPERATOR_IMAGE)"' config/manifests/bases/authorino-operator.clusterserviceversion.yaml
```

Comment:
I think line 325 will fail since we have 2 deployments in the manifests now, i.e., `authorino-operator` and `authorino-webhooks`.

Suggestion to change it to:

```mk
	yq ea -e 'select([.][].kind == "Deployment") | select([.][].metadata.name == "authorino-operator").spec.template.spec.containers[0].image | . == "$(OPERATOR_IMAGE)"' config/deploy/manifests.yaml
	yq ea -e 'select([.][].kind == "Deployment") | select([.][].metadata.name == "authorino-webhooks").spec.template.spec.containers[0].image | . == "$(DEFAULT_AUTHORINO_IMAGE)"' config/deploy/manifests.yaml
```

Reply:
I've updated the verification steps accordingly.